Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=dafb70ce1026d4d6ef1b16ad6996c9589bb11cce
Commit:     dafb70ce1026d4d6ef1b16ad6996c9589bb11cce
Parent:     f84e3e915ea03dfa6e32626fc25a4f284ef222ac
Author:     Giuseppe Bilotta <giuseppe.bilotta@gmail.com>
AuthorDate: Tue Jan 13 08:58:49 2009 -0500
Committer:  Takashi Iwai <tiwai@suse.de>
CommitDate: Tue Jan 13 15:14:12 2009 +0100

    ALSA: hda - Add quirk for another HP dv5
    
    Add the model=hp-m4 quirk for another HP dv5 (103c:3603)
    Reference: kernel bug#12440
    	http://bugzilla.kernel.org/show_bug.cgi?id=12440
    
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Cc: stable@kernel.org

 sound/pci/hda/patch_sigmatel.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/sound/pci/hda/patch_sigmatel.c b/sound/pci/hda/patch_sigmatel.c
index a4e2d8f..9acf2f0 100644
--- a/sound/pci/hda/patch_sigmatel.c
+++ b/sound/pci/hda/patch_sigmatel.c
@@ -1736,6 +1736,8 @@ static struct snd_pci_quirk stac92hd71bxx_cfg_tbl[] = {
 		      "HP dv7", STAC_HP_M4),
 	SND_PCI_QUIRK(PCI_VENDOR_ID_HP, 0x30fc,
 		      "HP dv7", STAC_HP_M4),
+	SND_PCI_QUIRK(PCI_VENDOR_ID_HP, 0x3603,
+		      "HP dv5", STAC_HP_M4),
 	SND_PCI_QUIRK(PCI_VENDOR_ID_HP, 0x361a,
 				"unknown HP", STAC_HP_M4),
 	SND_PCI_QUIRK(PCI_VENDOR_ID_DELL, 0x0233,
